src/common/configuration/ConfigService.js

```diff
@@ -126,6 +126,19 @@
       if (goog.isDefAndNotNull($window.config)) {
         goog.object.extend(this.configuration, $window.config, {});
       }
+      
+      // proxy basemap if config value proxyBaseMap is true
```

Comment:
Basemaps should have use_proxy and therefore be proxied due to: https://github.com/boundlessgeo/geonode/pull/170/files#diff-aadb2492556667858a9abd91a23ef330R485

However, MapLoom is constructing a default basemap hence this code is required, since GeoNode won't have any knowledge or access to it.